features/pakiti/client/configv3.pan

```diff
@@ -0,0 +1,37 @@
+unique template features/pakiti/client/configv3;
+
+variable PAKITI_RPMS ?= if_exists('config/pakiti/client/config');
```

Comment:
any relation to https://github.com/quattor/configuration-modules-core/blob/master/ncm-metaconfig/src/main/metaconfig/pakiti/pan/schema.pan?